### DISCUSSION TITLE
How to close this tab via keymap without wreaking havoc?

### DISCUSSION THREAD

**Discussion by borasemiz** [LazyVim/LazyVim#2679] [Q&A] [answered]
Hey. Yet another neovim noob. I installed LazyVim and I have bufferline and neotree extension seem to play nicely. I would like to close my currently working tab/window/buffer (I don't know which one is correct name), I can close it using the mouse click but I want to do it via keyboard. How to do this?

![image](https://github.com/LazyVim/LazyVim/assets/10868750/ea046400-e927-418e-acf5-4b2c936e1c21)

I've already tried couple of commands:

- `:bd`: It closes some thing but the layout of neotree changes. After closing via this command, neotree spans the entire terminal screen.
- `:quit` or `CTRL-W q`: It closes the neotree window, but not the current tab/window/buffer. Applying this command again quits neovim altogether (as expected).

**Comment by folke** [comment, marked as answer]
`<leader>bd`
